docs/examples/ex13.py

```diff
@@ -40,10 +40,10 @@
 basis = Basis(mesh, elements)
 A = asm(laplace, basis)
 
-boundary_dofs = basis.find_dofs()
+boundary_dofs = np.concatenate([basis.get_dofs(b) for b in mesh.boundaries])
 
 u = basis.zeros()
-u[boundary_dofs['positive'].all()] = 1.
+u[basis.get_dofs("positive").all()] = 1.
```

Comment:
`basis.get_dofs(("positive", "ground")).all()`

Reply:
No, I don't think so?  In this example the Dirichlet boundary is the disjoint union of `"positive"` and `"ground"` but the imposed values are unity on one and zero on the other.  There is also a homogeneous Neumann boundary.  I think of the three in physical terms as source, sink, and insulated, respectively.  Here I just want to assign ones to `"positive"`, leaving zeros on `"ground"`.  Later when the all the Dirichlet degrees of freedom are constrained, it will be something like the suggestion.

Ah, I see, this suggestion should apply to 

https://github.com/kinnala/scikit-fem/blob/eb5a56562b19c3d525388f4a1298f78443ca35d1/docs/examples/ex13.py#L43

Ta.

Reply:
Adopted in 334bec5.